doc/project/project_roles.rst

```diff
@@ -314,15 +314,37 @@ Release Activity
 Merge Criteria
 ++++++++++++++
 
-* All continuous integration checks have passed
+* Minimal of 2 approvals:
+
+  * A collaborator from the same subsystem.
+  * Alternately another maintainer of another subsystem
+  * Approval by the assignee
+
+* Four eye principle on the organisation level.
+
+  * Changes or additions to common and shared code shall have reviews from
+    different organisations (at least one review and approval from a different
+    submitters' organisation).
+  * Changes or additions to hardware support (driver, SoC, boards) shall at
+    least have the merger be from a different organisation.
+  * Release engineers shall not merge code changes originating from the same
+    organisation if all other reviewers are also from the same organisation.
+    To be able to merge such changes, at least one review shall be from a
+    different organisation.
+  * Exceptions can be made for areas with contributions largly coming from one
```

Comment:
Are this exceptions to be done by the release managers as they go?
or can release managers get into trouble if they merge something on good faith/under the understanding there is appropriate reviews?

Reply:
I meant to imply that we should be clear about who makes the exception, i.e.
"Release engineers may make exceptions for areas with contributions largely coming from one"..

By the way, largly => largely

Sidecomment:

> When in doubt, just ask someone else to merge something you are not sure about.

When in doubt typically the merger moves to the next PR ;)